sdks/python/apache_beam/transforms/util.py

```diff
@@ -196,6 +199,16 @@ def _empty(actual):
   return _empty
 
 
+def contains_in_any_order(expected):
```

Comment:
Is the only difference between this and equal_to is that this requires countable elements and equal_to requires comparable elements? Seems a subtle distinction...